settings.gradle.kts

```
@@ -34,6 +36,10 @@ include(
     ":appyx-components:internal:test-drive:common",
     ":appyx-components:experimental:cards:android",
     ":appyx-components:experimental:cards:common",
+    ":appyx-components:modal:android",
+    ":appyx-components:modal:common",
+    ":appyx-components:modal:android",
+    ":appyx-components:modal:common",
```

These are duplicated
```suggestion
    ":appyx-components:modal:android",
    ":appyx-components:modal:common",
```

---

Done